api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -407,24 +407,31 @@ public void sendBillingSetupEmail(DbUser dbUser, SendBillingSetupEmailRequest em
 
   @Override
   public void sendEgressRemediationEmail(
```

Comment:
instead of passing the boolean to this method, can you instead create a new method sendVwbEgressRemediationEmail which is only called from EgressVwbRemediationService. You can still reuse the code in sendEgressRemediationEmail method by extracting it to a private method that accepts the boolean

Reply:
I considered that, but because vwb vs terra is a minor field difference, I feel I do not need to add one more public method but just use one boolean to control the placeholder value. Usually, I feel 2 seperate method is prefered when we have more diffs/logic
I can change to the other option if you feel we have more benefit of doing that

Reply:
yea I see. My feeling is that passing a boolean is always less readable than using a properly named method. If it's a lot of work let's keep it this way.